templates/tab/js/m365/package.json

```diff
@@ -4,15 +4,14 @@
   "private": true,
   "dependencies": {
     "@fluentui/react-northstar": "^0.62.0",
-    "@microsoft/mgt-element": "2.4.1-next.teamsfx.8d56785",
-    "@microsoft/mgt-react": "2.4.1-next.teamsfx.8d56785",
-    "@microsoft/mgt-teamsfx-provider": "2.4.1-next.teamsfx.8d56785",
+    "@microsoft/mgt-element": "^2.6.2",
+    "@microsoft/mgt-react": "^2.6.2",
+    "@microsoft/mgt-teamsfx-provider": "^2.6.2",
     "@microsoft/microsoft-graph-client": "^3.0.1",
     "@microsoft/teams-js": "^2.2.0",
-    "@microsoft/teamsfx": "2.0.0-beta.0",
-    "@microsoft/teamsfx-react": "2.0.0-beta.1",
+    "@microsoft/teamsfx": "^2.0.1-alpha",
+    "@microsoft/teamsfx-react": "^2.0.1-alpha",
     "axios": "^0.21.1",
-    "msteams-react-base-component": "^4.0.1",
```

Comment:
I remember we had discussed with the `msteams-react-base-component` author on teamsjs v2 support. It seems it's not opensource friendly if just copying code and getting rid of the original package. That's also why we kept `msteams-react-base-component` in the m365 tab template.

See related threads:
- https://github.com/wictorwilen/msteams-react-base-component/issues/28
- https://github.com/OfficeDev/TeamsFx/pull/4989

BTW, @MuyangAmigo - any update I missed for the `msteams-react-base-component` package?

Reply:
We have talked to Wictor about merging `msteams-react-base-component` into `TeamsFx-React`, and he was open to the merging plan as long as we were able to ensure that each hook can be imported separately. We also add supports for both ESM and CJS so that webpack and other tools can do proper tree-shaking and optimizations. Details of the merging can be seen here: https://github.com/OfficeDev/TeamsFx/pull/6362.

Therefore, we remove `msteams-react-base-component` from the templates.

Reply:
Cool, good to know that, THX!